critiquebrainz/ws/review/views.py

```diff
@@ -225,7 +228,13 @@ def review_modify_handler(review_id, user):
 
     **OAuth scope:** review
 
+    :json string text: Text part of review, min length is 25, max is 5000 **(optional)**
+    :json integer rating: Rating part of review, min is 1, max is 5 **(optional)**
+
+    **NOTE:** The value of unmodified parameter should be set to its value in previous revison.
+
     :statuscode 200: success
+    :statuscode 400: invalid request (see source)
```

Comment:
Nope. No one is going to go and see the source.

Reply:
Ok. Will remove it.